nixops/backends/__init__.py

```diff
@@ -419,27 +419,35 @@ def copy_closure_to(self, path):
             env=env,
         )
 
-    def get_scp_name(self):
+    def _get_scp_name(self) -> str:
```

Comment:
double-checked that this wasn't used in plugins. I probably wasn't conclusive, but I checked at least GCE, AWS, and nix-community's plugins.